local-development/docker-compose.yml

```diff
@@ -20,22 +20,19 @@ services:
         tag: fairspace-keycloak-postgres
 
   fairspace-keycloak:
-    image: jboss/keycloak:12.0.4
+    image: keycloak/keycloak:20.0.3
     container_name: fairspace-keycloak
     environment:
-      DB_VENDOR: POSTGRES
-      DB_ADDR: fairspace-keycloak-postgres
-      DB_DATABASE: keycloak
-      DB_USER: keycloak
-      DB_PASSWORD: keycloak
-      KEYCLOAK_HOSTNAME: ${KEYCLOAK_HOSTNAME:?Please configure KEYCLOAK_HOSTNAME.}
-      KEYCLOAK_HTTP_PORT: ${KEYCLOAK_PORT:-5100}
-      KEYCLOAK_IMPORT: /tmp/realm-export.json
-      KEYCLOAK_REALM: ${KEYCLOAK_REALM:?Please configure KEYCLOAK_REALM}
```

Comment:
this one was used mainly in the realm template, currently it's not set at all...